packages/seo-store/src/form/slice/keyphrases.js

```diff
@@ -74,13 +74,25 @@ const keyphrasesSlice = createSlice( {
 	},
 } );
 
-export const keyphrasesSelectors = {
-	selectKeyphrases: state => get( state, "form.keyphrases", {} ),
+const selectors = {
+	selectKeyphraseEntries: state => get( state, "form.keyphrases", {} ),
 	selectKeyphrase: ( state, id = FOCUS_KEYPHRASE_ID ) => get( state, `form.keyphrases.${ id }.keyphrase` ),
-	selectSynonyms: ( state, id = FOCUS_KEYPHRASE_ID ) => get( state, `form.keyphrases.${ id }.synonyms` ),
+	selectKeyphraseSynonyms: ( state, id = FOCUS_KEYPHRASE_ID ) => get( state, `form.keyphrases.${ id }.synonyms` ),
 	selectKeyphraseIds: ( state ) => keys( get( state, "form.keyphrases" ) ),
 };
 
+export const keyphrasesSelectors = {
```

Comment:
Is it necesarry to wrap these in a container again? Can't we just export all `selectK...` selector individually?

Reply:
This is for conveniency for the opposite reason: so we don't have to import all selectors individually in `src/index.js`. If you check that file you see this pattern also matches the actions pattern.